Week1/homework/index.js

```diff
@@ -0,0 +1,205 @@
+'use strict';
+
+const mysql = require('mysql');
+const express = require('express');
+const bodyParser = require('body-parser');
+const { promisify } = require('util');
+
+const app = express();
+app.use(bodyParser.json());
+
+const data = require('./cities.json');
+
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'root',
+  password: 'password',
+  database: 'world',
+});
+
+connection.connect();
+
+const createCountries = `CREATE TABLE countries (
+  Name varchar(254) ,
+  Continent varchar(50),
+  Region varchar(50) ,
+  SurfaceArea float(2) ,
+  IndepYear varchar(50) ,
```

Comment:
We could argue here with an int :)

Reply:
Okay, we will :)

Reply:
Table hasn't values Error: ER_TRUNCATED_WRONG_VALUE_FOR_FIELD: Incorrect integer value: ' null' for column 'IndepYear' at row 1 (When int)